### PR TITLE
CPDP-2583-fix-content

### PR DESCRIPTION
**Pull request by sam-miele**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -774,10 +774,10 @@ previousReturnExistsWithSameCompletionDate.amend.indirectDisposals.title=Indirec
 previousReturnExistsWithSameCompletionDate.amend.indirectDisposals.p1=You’ll need to amend the return that had shares with the same sale of disposal date as the disposal you’re wanting to report now.
 
 previousReturnExistsWithSameCompletionDate.title=You’ll need to change an existing return
-previousReturnExistsWithSameCompletionDate.p1=You already sent a return that had a disposal on <strong class="bold">{0}</strong>. You’ll need to change that return instead of creating a new one.
+previousReturnExistsWithSameCompletionDate.p1=You already have a return with completion date of <strong class="bold">{0}</strong>. You’ll need to change that return instead of creating a new one.
```
Comment (ugh92) [inline conf/messages:777]: You already sent a return that had a disposal with a completion date of [date month year].

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala [hunk 2]
```diff
@@ -82,6 +84,8 @@ class CommonTriageQuestionsController @Inject() (
 
   import CommonTriageQuestionsController._
 
+  val newCompletionDateUsedFlag = servicesConfig.getBoolean("amend-returns.enabled")
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala:87]: you don't need this flag here - you can use `viewConfig.amendReturnsEnabled`

File: conf/application.conf [hunk 3]
```diff
@@ -260,4 +260,5 @@ play.i18n {
     langs = ["en", "cy"]
 }
 
-amend-returns.enabled = true
\ No newline at end of file
+amend-returns.enabled = true
+new-completion-date-already-used-page-exit-flag.enabled = false
```
Comment (AndyHWChung) [inline conf/application.conf:264]: you don't need this new flag - the behaviour should be based on the amend returns enabled flag

File: conf/messages [hunk 4]
```diff
@@ -5088,11 +5086,8 @@ unmetDependency.x2.li2=work out the new amount of tax due
 unmetDependency.x2.li3=upload a document showing how the new amount of tax due was worked out
 
 disposalDateInDifferentTaxYear.title=You cannot change this return online
-disposalDateInDifferentTaxYear.p1=You must <a href="{0}">contact HMRC</a>.
-disposalDateInDifferentTaxYear.p2=This is because sales or disposals with the same completion date must all be in the same return.
-disposalDateInDifferentTaxYear.p3=Tell HMRC you need to change:
-disposalDateInDifferentTaxYear.li1=a previous return
-disposalDateInDifferentTaxYear.li2=the return that has the same disposal completion date as the one you entered on the previous page
+disposalDateInDifferentTaxYear.p1=This is because the contract exchange date you entered would change the tax year the disposal was in.	disposalDateInDifferentTaxYear.p1=You must <a href="{0}">contact HMRC</a>.
```
Comment (AndyHWChung) [inline conf/messages:5089]: looks like you need a new line here to separate the messages

Comment (AndyHWChung) [reply conf/messages:5089]: though they have the same key, so maybe it's just choosing the correct version

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**